code/modules/food_and_drinks/recipes/recipes_microwave.dm

```diff
@@ -476,10 +476,141 @@
 		/obj/item/reagent_containers/food/snacks/meat,
 		/obj/item/reagent_containers/food/snacks/cheesewedge,
 		/obj/item/reagent_containers/food/snacks/boiledegg,
-		/obj/item/reagent_containers/food/snacks/grown/tomato
+		/obj/item/reagent_containers/food/snacks/tomatoslice
 	)
 	result = /obj/item/reagent_containers/food/snacks/superbiteburger
 
+/datum/recipe/microwave/crazyburger
+	reagents = list("cornoil" = 15)
+	items = list(
+		/obj/item/reagent_containers/food/snacks/bun,
+		/obj/item/reagent_containers/food/snacks/meat,
+		/obj/item/reagent_containers/food/snacks/meat,
+		/obj/item/reagent_containers/food/snacks/cheesewedge,
+		/obj/item/reagent_containers/food/snacks/cheesewedge,
+		/obj/item/reagent_containers/food/snacks/grown/cabbage,
+		/obj/item/reagent_containers/food/snacks/grown/chili,
+		/obj/item/toy/crayon/green,
+		/obj/item/flashlight/flare
+	)
+	result = /obj/item/reagent_containers/food/snacks/crazyburger
+
+/datum/recipe/microwave/blt
+	items = list(
+		/obj/item/reagent_containers/food/snacks/breadslice,
+		/obj/item/reagent_containers/food/snacks/breadslice,
+		/obj/item/reagent_containers/food/snacks/grown/cabbage,
+		/obj/item/reagent_containers/food/snacks/tomatoslice,
+		/obj/item/reagent_containers/food/snacks/bacon
+	)
+	result = /obj/item/reagent_containers/food/snacks/blt
+
+/datum/recipe/microwave/peanut_butter_jelly
+	reagents = list("cherryjelly" = 5, "peanutbutter" = 5)
```

Comment:
This is more of a request than a review, but I'd love if we could make these with slime jelly too, like other jelly foods

Reply:
I could do that, sure, it wouldn't be any extra effort really.